contracts/GenArt721CoreV3.sol

```diff
@@ -162,30 +160,31 @@ contract GenArt721CoreV3 is ERC721Enumerable, IGenArt721CoreContractV3 {
         internal
         returns (uint256 _tokenId)
     {
-        uint256 tokenIdToBe = (_projectId * ONE_MILLION) +
+        uint256 nextTokenId = (_projectId * ONE_MILLION) +
             projects[_projectId].invocations;
 
         projects[_projectId].invocations = projects[_projectId].invocations + 1;
 
-        bytes32 hash = keccak256(
+        bytes32 tokenHash = keccak256(
             abi.encodePacked(
                 projects[_projectId].invocations,
                 block.number,
                 blockhash(block.number - 1),
                 randomizerContract.returnValue()
             )
         );
-        tokenIdToHash[tokenIdToBe] = hash;
-        hashToTokenId[hash] = tokenIdToBe;
 
-        _mint(_to, tokenIdToBe);
+        tokenIdToHash[nextTokenId] = tokenHash;
+        hashToTokenId[tokenHash] = nextTokenId;
```

Comment:
I know it's been here since the beginning, but I never understood the use case - why do we need a hash to token mapping?

Reply:
I also support removing this due to the extra storage gas we are adding on mint. I am not aware of any important use-cases, and our subgraph can easily perform this type of lookup.

Reply:
I don't know of any use case for this, but initially left just to not break backwards compatibility given that this is exposed publicly. That said, if we don't imagine it having any purpose we should definitely just strip it for V3. @ryley-o WDYT?

Reply:
updating at same time 😆 see comment above - let's remove.
Cc @lyaunzbe in case he is aware of anything using this in the pipeline

Reply:
SGTM–I'll strip it :)

Reply:
👍 shouldn't be in use anywhere on the pipeline side